src/hid_manager.rs

```diff
@@ -0,0 +1,42 @@
+use libc::c_void;
+use cf::{CFAllocatorRef, CFDictionaryRef, CFIndex};
+use types::IOOptionBits;
+use io_return::IOReturn;
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDManager {
+    __private: c_void,
+}
+pub type IOHIDManagerRef = *const __IOHIDManager;
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDDevice {
+    __private: c_void,
+}
+pub type IOHIDDeviceRef = *const __IOHIDDevice;
+
+pub type IOHIDDeviceCallback = extern fn(context: *mut c_void, result: IOReturn, sender: *mut c_void, device: IOHIDDeviceRef);
+
+pub type IOHIDReportCallback = extern fn(context: *mut c_void, result: IOReturn, sender: *mut c_void, report_type: IOHIDReportType, 
+                                         report_id: u32, report: *mut u8, report_len: CFIndex);
+
+pub type IOHIDReportType = u32;
```

Comment:
The C definition is a plain enum. I think the equivalent type in Rust would be `c_int`.